doc/en/components/grids/_shared/toolbar.md

```diff
@@ -10,204 +10,44 @@ sharedComponents: ["Grid", "TreeGrid", "HierarchicalGrid"]
-    <igx-grid-toolbar>
-    </igx-grid-toolbar>
-</igx-hierarchical-grid>
-```
-}
 
 For a comprehensive look over each of the default UI components, continue reading the **Features** section
 below.
 
-@@if (igxName === 'IgxHierarchicalGrid') {
-
-## Toolbar with child grids
-Due to certain limitations in how the child grids of an IgxHierarchicalGrid are implemented and how DI scope works, there is a caveat when
-using the toolbar in the scope of child grids. When defining a toolbar component inside the `igx-row-island` tags, always make sure
-to use the IgxGridToolbar directive on the toolbar itself and pass the provided grid instance as an input property to the toolbar itself.
-This will make sure you always have the correct grid instance in the scope of your template:
-
-```html
-<igx-hierarchical-grid>
-    ...
-    <igx-row-island>
-        <!--
-            You can name the binding from igxGridToolbar however you want. Just make sure to use
-            it inside the template if you need to access the grid instance.
-        -->
-        <igx-grid-toolbar [grid]="gridRef" *igxGridToolbar="let gridRef">
-            <igx-grid-toolbar-title>Child toolbar {{ gridRef.parentIsland.level }}</igx-grid-toolbar-title>
-        </igx-grid-toolbar>
-    </igx-row-island>
-    ...
-</igx-hierarchical-grid>
-```
-
-}
-
 ## Features
 
 The toolbar is great at separating logic/interactions which affects the grid as a whole.
 As shown above, it can be configured to provide default components for controlling, column hiding, column pinning,
 advanced filtering and exporting data from the grid.
 These features can be enabled independently from each other by following a pattern similar to the card component of
 the {ProductName} for {Platform} suite.
-Listed below are the main features of the toolbar with example code for each of them.
-
-
-<code-view style="height:630px"
-           data-demos-base-url="{environment:dvDemosBaseUrl}"
-           iframe-src="{environment:dvDemosBaseUrl}/{ComponentSample}-toolbar-sample-2" >
-</code-view>
-
 
 ### Title
 Setting a title for the toolbar in your grid is achieved by using the `GridToolbarTitleDirective`.
 Users can provide anything from simple text to more involved templates.
-```html
-<igx-grid-toolbar>
-    <igx-grid-toolbar-title>Grid toolbar title</igx-grid-toolbar-title>
-</igx-grid-toolbar>
-```
-
-### Actions
-The `GridToolbarTitleDirective` exposes a container where users can place actions/interactions in relation to the parent grid.
-As with the title portion of the toolbar, users can provide anything inside that template part, including the default
-toolbar interaction components.
-```html
-<igx-grid-toolbar>
-    <igx-grid-toolbar-actions>
-        <button igxButton>Action</button>
-        <igx-select></igx-select>
-        ...
-    </igx-grid-toolbar-actions>
-</igx-grid-toolbar>
+```Razor
+<IgbDataGrid>
+    <IgbDataGridToolbar title="CustomTitle" />
+</IgbDataGrid>
```

Comment:
why was this angular code removed from topic?

Reply:
I had conflicts and seems to have removed extra content. I have resolved them again, please check now